.circleci/config.yml

```diff
@@ -189,6 +208,7 @@ workflows:
   unit_tests:
     jobs:
       - tests_local_browsers
+      - browserstack_tests
```

Comment:
Can we do `tests_browserstack`. So the naming convention is consistent

Reply:
Good point. Will update that